src/js/components/Paragraph/stories/Theme.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Paragraph } from 'grommet';
+import { deepMerge } from 'grommet/utils';
+import { grommet } from 'grommet/themes';
+
+const customTheme = deepMerge(grommet, {
+  paragraph: {
+    font: {
+      family: 'Comic Sans MS',
+    },
+  },
+});
+
+const All = () => (
+  <Grommet theme={customTheme}>
+    <Paragraph>
+      The font family for this paragraph is being defined by a custom theme.
+    </Paragraph>
+  </Grommet>
+);
+
+storiesOf('Paragraph', module).add('Theme', () => <All />, {
+  chromatic: { disable: true },
```

Comment:
why do you think it is OK to exclude the test from chromatic?

Reply:
The customTheme font is showing up in storybook but the chromatic snapshots are not capturing the font.

> Browsers can decide to render HTML in multiple passes when custom fonts are used. They do this to speed up the time-to-first-meaningful-paint.
> Unfortunately, this behavior can cause your story to render without the custom font.

found information about this here: https://www.chromatic.com/docs/resource-loading#loading-custom-fonts

Reply:
Hello

Reply:
Test